mne/decoding/search_light.py

```diff
@@ -338,7 +358,10 @@ def _sl_score(estimators, X, y):
     """
     n_iter = X.shape[-1]
     for ii, est in enumerate(estimators):
-        _score = est.score(X[..., ii], y)
+        if scoring is not None:
+            _score = scoring(est, X[..., ii], y)
```

Comment:
@agramfort this is a single/multi class scoring problem. Isn't there a way to handle thus by sklearn scorers?